_pages/install/integrations/open-payment-framework-integration.md

```diff
@@ -0,0 +1,150 @@
+---
+title: SAP Open Payment Framework Integration
+feature:
+  - name: Open Payment Framework Integration
+    spa_version: [NOT_SPECIFIED_YET]
+    cx_version: 2211
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} [NOT_SPECIFIED_YET] {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The Open Payment Framework provides merchants with a streamlined way to connect their chosen Payment Service Provider using configurations.
+
+For more information, see [Open Payment Framework in a Nutshell](https://help.sap.com/docs/SAP_UPSCALE_COMMERCE/0160c41e0de84b218d05bc1185213d1d/5efc3463b4504d27bb9c4fbbb95a4ccc.html?locale=en-US#open-payment-framework-in-a-nutshell) on the SAP Help Portal.
+
+***
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+***
+
+## Requirements
+
+To integrate SAP Open Payment Framework with Composable Storefront, you must have SAP Commerce Cloud 2211 or newer.
+
+## Composable Storefront Integration with Payment Service Provider
+
+The Composable Storefront offers a versatile integration with the Open Payment Framework. Depending on the configurations set within the Open Payment Framework, the Composable Storefront can adapt its behavior to present different user experience (UX) patterns during the payment process.
+
+### UX Patterns
+
+Here are the distinct UX patterns that the Composable Storefront can render based on the configurations:
+
+- **Gateway Page**: In this mode, consumers are redirected to a page that is hosted by the Payment Service Provider. Consumers finalize their payment on the trusted platform of the Payment Service Provider, ensuring a secure transaction experience.
+
+- **iFrame**: This pattern involves displaying an embedded iframe on the page.
+  It allows consumers to complete their payment within the iframe without navigating away from the main site.
+
+- **Hosted Fields**: The pattern where hosted fields are displayed directly on the page. Consumers can finish their payment seamlessly within these hosted fields, offering a streamlined experience without redirection.
+
+#### Support for 3DS
+
+It is important to note that all the above UX patterns support 3DS (3-D Secure) authentication. This means that consumers will benefit from an additional layer of security during the payment process, regardless of which UX pattern they encounter.
+
+## Backend Configuration
+
+### Sample data
+
+Certain features of OPF, including the payment and review page or the call-to-action scripts, necessitate specific sample data configurations on the backend.
+
+If the OPF features do not function correctly, import the following impex script into the backend system:
+
```

Comment:
Here, we're saying "if the OPF features do not function correctly", but at this point in the documentation, the opf lib hasn't been installed yet... Is this information that should go in a troubleshooting section later in the doc, or do customers always need to run this impex for opf to work, and I should be just rephrasing this here to say, you need to run this impex for your OPF features to work properly?....... 🙂

Reply:
Exactly, customers must execute this impex for OPF to operate correctly. If they encounter issues after installing the OPF library and find certain functionalities missing, they should verify that they've imported the necessary impex scripts.